### DISCUSSION TITLE
Rubric on Canvas Not Matching This Week's Assignment Repo

### DISCUSSION THREAD

**Discussion by coopsand** [USU-ITLS-x265-23-summer/class-repo#109] [Q&A] [answered]
I was just reviewing the rubric in canvas for this weeks assignment (pictured below), and I don't see any readings or videos on this weeks page. Did I just happen to miss the links somewhere? I have completed what was outlined in the repo - I just don't want to miss anything.

<img width="844" alt="Screenshot 2023-06-05 at 10 51 12 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133150498/51e63ce6-0468-4ebf-9822-9fef730f1367">

**Comment by jakegros** [comment, marked as answer]
I didn't see any other video/reading either.  I think the safest bet is to follow the repo.  It makes sense that it would be more current as @metasean moved the focus of the class flow to GitHub.  I assume Canvas wasn't updated to match.  🤷‍♂️

**Comment by derekscholes** [comment]
This honestly might be from the week 6 rubric. Looking at week 6 they appear to be the same. However, the week assignments for week 6 match up with this rubric. I would follow the instructions on the repo.
<img width="599" alt="Screenshot 2023-06-09 at 9 49 10 AM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132800749/5d3f2ac9-5915-4553-a62a-7c0d360ca0cf">
